scripts/signedbuildskadi.sh

```diff
@@ -1,274 +1,311 @@
+  sudo sed -i "s/UPLOAD_ENABLED = False/UPLOAD_ENABLED = True/g" /etc/timesketch.conf
+  sudo sed -i "s/GRAPH_BACKEND_ENABLED = False/GRAPH_BACKEND_ENABLED = True/g" /etc/timesketch.conf
+  sudo sed -i "s#@localhost/timesketch#@postgres/timesketch#g" /etc/timesketch.conf
+  sudo sed -i "s/ELASTIC_HOST = '127.0.0.1'/ELASTIC_HOST = 'elasticsearch'/g" /etc/timesketch.conf
+  sudo sed -i "s@'redis://127.0.0.1:6379'@'redis://redis:6379'@g" /etc/timesketch.conf
+  sudo sed -i "s/NEO4J_HOST = '127.0.0.1'/NEO4J_HOST = 'neo4j'/g" /etc/timesketch.conf
+
+  # Setup Nginx Auth
+  echo $NGINX_PASSWORD | sudo htpasswd -i -c /etc/nginx/.skadi_auth $NGINX_USER
+}
+
+clean_APT () {
+  # Clean APT
+  sudo apt-get -y autoremove --purge
+  sudo apt-get -y clean
+  sudo apt-get -y autoclean
+}
+
+containers_up () {
+  # Deploy the Skadi solution defined in ./docker-compose.yml
+  sudo docker-compose up -d
+
+  # Create a template in ES that sets the number of replicas for all indexes to 0
+  echo "Waiting for ElasticSearch service to respond to requests"
+  until $(curl --output /dev/null --silent --head --fail http://localhost:9200); do
+      printf '.'
+      sleep 5
+  done
+  echo "Setting the ElasticSearch default number of replicas to 0"
+
+  curl -XPUT 'localhost:9200/_template/number_of_replicas' \
+      -d '{"template": "*","settings": {"number_of_replicas": 0}}' \
+      -H'Content-Type: application/json'
+
+  echo "Waiting for Kibana service to respond to requests"
+  until $(curl --output /dev/null --silent --head --fail http://localhost:5601); do
+      printf '.'
+      sleep 5
+  done
+
+  echo "Importing Saved Objects to Kibana and setting default index"
+  curl -X POST "http://localhost:5601/api/saved_objects/_bulk_create" -H 'kbn-xsrf: true' -H 'Content-Type: application/json' --data-binary @/opt/Skadi/objects/kibana_6.x_cli_import.json
+  curl -X POST "http://localhost:5601/api/kibana/settings/defaultIndex" -H "Content-Type: application/json" -H "kbn-xsrf: true" -d '{"value": "06876cd0-dfc5-11e8-bc06-31e345541948"}'
+}
+
+ensure_TS_up () {
+# The TimeSketch container needs to be running before continuing and this
+# requires the other containers to be up and running too. This can take time
+# so this loop ensures all the parts are running and timesketch is responding
+# to web requets before continuing
+sudo docker restart timesketch
+echo "Waiting 10 seconds for TimeSketch to become available"
+sleep 10
+echo "Press CTRL-C at any time to stop installation"
+until $(curl --output /dev/null --silent --head --fail http://localhost/timesketch); do
+    echo "No response, restarting the TimeSketch container and waiting 10 seconds to try again"
+    sudo docker restart timesketch
+    sleep 10
+done
+echo "TimeSketch available. Continuing"
+}
+
+grafana_setup () {
+  # Install Grafana for Monitoring
+  git clone https://github.com/orlikoski/skadi_dockprom.git

```

Comment:
This is no longer needed if using git cone --recursive

Reply:
removed this and updated